environments/local-rpm-repo/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM centos:7
+RUN yum install -y epel-release && \
+    yum install -y \
+    createrepo \
+    nginx \
+    && \
+    yum clean all
+
+RUN mkdir -p /var/www/html/repos/{base,centosplus,extras,updates}
+COPY docker-ce.repo /etc/yum.repos.d/docker-ce.repo
+COPY kubernetes.repo /etc/yum.repos.d/kubernetes.repo
+RUN yum --downloadonly --downloaddir /var/www/html/repos install -y --disableexcludes=kubernetes \
+    device-mapper-persistent-data \
+    docker-ce-19.03.8 \
+    kubeadm-1.16.11 \
```

Comment:
is it possible to install the same package of different versions?

Reply:
Yes; there's some tradeoff in image size, so adding versions we don't use can slow down CI.
Suggest we go with the ones we need for now and you can build your own image locally to see how things change.